libs/server/Resp/Parser/RespCommand.cs

```diff
@@ -1902,9 +1902,7 @@ private void HandleAofCommitMode(RespCommand cmd)
             if (dcurr == networkSender.GetResponseObjectHead())
                 waitForAofBlocking = false;
 
-            // If we are in NetworkSkip's state we do not need to be changing the AOF blocking flag
```

Comment:
please add a comment here for why this if is needed and correct.

Reply:
In particular, we need to make sure the Commit of the transaction (e.g., EXEC) will not respond until a commit is successfully done, if we are blocking on AOF.